src/app/pages/pdf/pdf.ts

```diff
@@ -0,0 +1,89 @@
+import { Component } from '@angular/core';
+import {
+  IonicPage,
+  NavController,
+  LoadingController,
+  NavParams,
+} from 'ionic-angular';
+import { PdfService } from 'old/src/app/services/pdf/pdf.service';
+import { AnalyticsService } from 'src/app/services/analytics/analytics.service';
+import { EventsService } from 'src/app/services/events/events.service';
+
+/**
+ * Generated class for the PdfPage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage({
+  name: 'pdf',
+  segment: 'facsimile/pdf/:facsimileId',
+})
+@Component({
+  selector: 'page-pdf',
+  templateUrl: 'pdf.html',
+})
+export class PdfPage {
+  private loading: any;
+  public pdf: string;
+  public facsimileId: string;
+  public collectionId: string;
+  public pdfFile: string;
+  public page = 1;
+  public title: string;
+  public pagesLoaded: 0 | undefined;
+
+  constructor(
+    private navCtrl: NavController,
+    private loadingCtrl: LoadingController,
+    private params: NavParams,
+    public pdfService: PdfService,
+    private events: EventsService,
+    private analyticsService: AnalyticsService
+  ) {
+    this.collectionId = this.params.get('collectionId');
```

Comment:
Check out from other ported pages how params work currently (in ngOnInit lifecycle events)